bors.toml

```diff
@@ -1 +1 @@
-status = ["continuous-integration/travis-ci/push"]
+status = ["build / Build and Test (push)"]
```

Comment:
Just guessing, but maybe just

```suggestion
status = ["build / Build and Test"]
```

The thing in parens is just the trigger (see in the PR checks where it is `(pull request)`.

Reply:
If https://www.christopherbiscardi.com/integrating-bors-with-github-actions is to believe, I think it should be

```suggestion
status = ["Build and Test"]
```

Reply:
Lets see

Reply:
That was it

Reply:
is this back to before or is github's ui tricking me?